privatek8s.tf

```
@@ -101,6 +101,45 @@ resource "azurerm_kubernetes_cluster_node_pool" "infracipool" {
   tags = local.default_tags
 }
 
+resource "azurerm_kubernetes_cluster_node_pool" "highmempool" {
+  name                  = "highmemlinux"
+  vm_size               = "Standard_D8s_v3"
+  os_disk_type          = "Ephemeral"
+  os_disk_size_gb       = 100
+  kubernetes_cluster_id = azurerm_kubernetes_cluster.privatek8s.id
+  enable_auto_scaling   = true
+  min_count             = 0
+  max_count             = 3
+  zones                 = [3]
+  vnet_subnet_id        = data.azurerm_subnet.privatek8s_tier.id
+  node_taints = [
+    "os=linux:NoSchedule",
+    "profile=highmem:NoSchedule",
+  ]
+
+  tags = local.default_tags
+}
+
+resource "azurerm_kubernetes_cluster_node_pool" "windowspool" {
+  name                  = "win"
```

```suggestion
resource "azurerm_kubernetes_cluster_node_pool" "windows2019pool" {
  name                  = "windows2019"
```

discussed together: using `2019` to ensure we can add a `2022` soon (without replacing this one)